### PR TITLE
MSI password boxes are not fully reset

### LINKED ISSUE #191: MSI - password boxes are not fully reset
**Found in version**
- 6.3.0-bc2

**Steps to reproduce**
- Enter password(s) in Service or X-Pack page (for existing service user / elastic users in Trial license)
- Click reset button for the respective page
- Select options to show password boxes again

**Expected result**
- The password boxes are empty

**Actual result**
- There are still bullets in the box, but the box value is empty (validation errors “password required”)

### PR DESCRIPTION
**Pull request by codebrain**
MSI password boxes are not fully reset

Closes #191

### CODE REVIEW

File: src/Installer/Elastic.Installer.UI/Elasticsearch/Steps/XPackView.xaml.cs [hunk 1]
```diff
@@ -92,6 +92,33 @@ protected override void InitializeBindings()
 					this.SkipPasswordGenerationCheckBox.IsEnabled = securityEnabled && canGenerateUsers;
 					
 				});
+
+			this.WhenAnyValue(view => view.ViewModel.ElasticUserPassword)
```
Comment (Mpdreamz) [inline src/Installer/Elastic.Installer.UI/Elasticsearch/Steps/XPackView.xaml.cs]: You can use `this.WhenAnyValue(x,y,z)` to subscribe to all these at once and have a single `Subscribe`

File: src/Installer/Elastic.Installer.UI/Elasticsearch/Steps/XPackView.xaml.cs [hunk 2]
```diff
@@ -92,6 +92,33 @@ protected override void InitializeBindings()
...
+				.Subscribe(x =>
+				{
+					if (x == XPackModel.DefaultElasticUserPassword)
```
Comment (Mpdreamz) [inline src/Installer/Elastic.Installer.UI/Elasticsearch/Steps/XPackView.xaml.cs]: Single line `if`

**Comment by codebrain** [comment]
@Mpdreamz - Addressed PR comments

**Comment by codebrain** [comment]
Backported to `6.x` and `6.3`

**Pull request merged**